tests/linearGaussian_snl_test.py

```diff
@@ -48,9 +46,9 @@ def test_snl_on_linearGaussian_api(num_dim: int):
 
 
 @pytest.mark.slow
-@pytest.mark.parametrize("num_dim", (1, 3))
+@pytest.mark.parametrize("num_dim", (1, 2))
```

Comment:
Is this about computational speed?

Reply:
more about the algorithm not coming close to 0.5 in 3D. And I thought the tests were for testing in 1D and multi D so that 2D is enough?

Reply:
Yes, 2D is fine, I was just curious :)